.changeset/shaggy-drinks-hammer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-org': patch
+---
+
+Display only first 50 group members
```

Comment:
This probably needs changing now to reflect the change has pagination etc :)

Reply:
yes, Fixed